chain/default_store.go

```diff
@@ -113,10 +114,15 @@ func (store *DefaultStore) Load(ctx context.Context) error {
 		if err != nil {
 			return errors.Wrap(err, "failed to add validated block to TipSet")
 		}
+		startHeight = blk.Height
 	}
+	logStore.Infof("start loading chain at tipset: %s, height: %d", tipCids.String(), startHeight)
```

Comment:
If having the height isn't important then we could remove the added variable above.

Reply:
i like having it, really helpful